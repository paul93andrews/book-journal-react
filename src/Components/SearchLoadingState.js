import React from 'react';

const SearchLoadingState = () => (
    <div>
        <h3>It's loading...</h3>
    </div>
);

export default SearchLoadingState;