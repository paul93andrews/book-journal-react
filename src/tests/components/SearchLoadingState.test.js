import React from 'react';
import { shallow } from 'enzyme';
import SearchLoadingState from '../../components/SearchLoadingState';

test('should render loading state correctly', () => {
    const wrapper = shallow(<SearchLoadingState />);
    expect(wrapper).toMatchSnapshot();
});