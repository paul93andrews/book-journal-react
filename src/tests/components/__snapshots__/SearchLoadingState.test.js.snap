// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render loading state correctly 1`] = `
<div>
  <h3>
    It's loading...
  </h3>
</div>
`;
